ext/BraketSimulatorPythonExt/BraketSimulatorPythonExt.jl

```diff
@@ -1,6 +1,10 @@
 module BraketSimulatorPythonExt
 
-using BraketSimulator, BraketSimulator.Braket, PythonCall, BraketSimulator.Dates
+using PrecompileTools
+
+@recompile_invalidations begin
+    using BraketSimulator, BraketSimulator.Braket, PythonCall, BraketSimulator.Dates
+end
```

Comment:
Why move into this block?

Reply:
Look at the PrecompileTools.jl documentation for an explanation.